Makefile

```diff
@@ -345,9 +344,24 @@ endif
 CLUSTER_OP_ANSIBLE_REPO   ?= https://github.com/openshift/openshift-ansible.git
 CLUSTER_OP_ANSIBLE_BRANCH ?= master
 
+define build-cluster-operator-ansible-image #(repo, branch, imagename, tag)
+	echo "repo $1"
+	echo "branch $2"
+	echo "image $3:$4"
+	docker build -t "$3:$4" --build-arg=CO_ANSIBLE_URL=$1 --build-arg=CO_ANSIBLE_BRANCH=$2 build/cluster-operator-ansible
+endef
+
 cluster-operator-ansible-image: build/cluster-operator-ansible/Dockerfile build/cluster-operator-ansible/playbooks/cluster-api-prep/deploy-cluster-api.yaml build/cluster-operator-ansible/playbooks/cluster-api-prep/files/cluster-api-template.yaml build/cluster-operator-ansible/playbooks/cluster-operator/node-config-daemonset.yml
-	docker build -t $(CLUSTER_OPERATOR_ANSIBLE_IMAGE) --build-arg=CO_ANSIBLE_URL=$(CLUSTER_OP_ANSIBLE_REPO) --build-arg=CO_ANSIBLE_BRANCH=$(CLUSTER_OP_ANSIBLE_BRANCH) build/cluster-operator-ansible
-	docker tag $(CLUSTER_OPERATOR_ANSIBLE_IMAGE) $(CLUSTER_OPERATOR_ANSIBLE_MUTABLE_IMAGE)
+	$(call build-cluster-operator-ansible-image,$(CLUSTER_OP_ANSIBLE_REPO),"master",$(CLUSTER_OPERATOR_ANSIBLE_IMAGE_NAME),$(VERSION))
+	docker tag $(CLUSTER_OPERATOR_ANSIBLE_IMAGE_NAME):$(VERSION) $(CLUSTER_OPERATOR_ANSIBLE_IMAGE_NAME):$(MUTABLE_TAG)
+
+
+cluster-operator-ansible-versions:
+	# build v3.9 on openshift-ansible:release-3.9
+	$(call build-cluster-operator-ansible-image,$(CLUSTER_OP_ANSIBLE_REPO),"release-3.9",$(CLUSTER_OPERATOR_ANSIBLE_IMAGE_NAME),"v3.9")
+
+	# build v3.10 on openshift-ansible:master
+	$(call build-cluster-operator-ansible-image,$(CLUSTER_OP_ANSIBLE_REPO),"master",$(CLUSTER_OPERATOR_ANSIBLE_IMAGE_NAME),"v3.10")
```

Comment:
Getting closer, I would kind of like any normal make images invocation to result in building both of these, either cluster-operator-ansible:v3.9 + v3.10, or v3.0-canary v3.10-canary. I'm hoping to have a default devel deploy have prepopulated ClusterVersions for both with a PR I'm working on now.

So in theory just a Make target cluster-operator-ansible-images that replaces these.

Would this be reasonable to do or any gotchas I'm not thinking of?